File: Data Structures/Algoritmos_Sort_java/Nodo.java
package estructuradedatos;

public class Nodo {
    public int info;
    public Nodo Sig;

        public Nodo()
        {
            info = 0;
            this.Sig = null;
        }
        public Nodo(int info)
        {
            this.info = info;
            this.Sig = null;
        }

        public Nodo(Nodo x)
        {
            this.info = x.info;
            this.Sig = x.Sig;
        }

    @Override
        public String toString()
        {
            String imprimir = ("[" + info + "]");
            return imprimir;
        }
}

File: Data Structures/Algoritmos_Sort_C++/Nodo.cpp
#include <iostream>
#include "Nodo.h"
#include <sstream>
using namespace std;

Nodo::Nodo(int dato) {
	info = dato;
	sig = NULL;
}

Nodo::Nodo() {
	info = -1;
	sig = NULL;
}

Nodo::~Nodo() {
	cout << "[" << info << "]" << " ha sido destruido" << endl;
}